src/telemetry/classification.ts

```diff
@@ -55,7 +55,10 @@ export interface IGlobalMetrics {
 }
 
 interface IRPCOperationClassification {
-  performance: { classification: 'SystemMetaData'; purpose: 'PerformanceAndHealth' };
+  [key: string]: {
```

Comment:
The GDPR classification tool doesn't support generic keys unfortunately--that's why I'd moved it to an array initially

Reply:
We should be able to delete the arbitrary string key here so that everything gets fully typed.